app/code/Magento/Backend/view/adminhtml/layout/adminhtml_dashboard_index.xml

```diff
@@ -10,9 +10,9 @@
     <body>
         <referenceContainer name="page.main.actions">
             <block class="Magento\Backend\Block\Store\Switcher" name="store_switcher" as="store_switcher" template="Magento_Backend::store/switcher.phtml">
-                <action method="setUseConfirm">
-                    <argument name="params" xsi:type="string">0</argument>
-                </action>
+                <arguments>
+                    <argument name="is_using_iframe" xsi:type="string">0</argument>
+                </arguments>
```

Comment:
Please explain why you did it?

Reply:
This variable not set anywhere in code-base so i think we should declare it here

Reply:
https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Backend/Block/Store/Switcher.php#L614
But it called in this class

Reply:
Add default value here. Other modules core or 3rd-party providers extensions can easy override it

Reply:
setUseConfirm is unneccesary because it's already set via Block

Reply:
It feels like it supposes to change the following flag: 
https://github.com/magento/magento2/pull/32434/files#diff-5c07e420bed1b6e2edda1d6165e1ca2dc4eacaab999bf149eaffa4f53c3823afR47